src/main/java/at/gammastrahlung/monopoly_server/game/gameboard/GameBoard.java

```diff
@@ -776,4 +776,13 @@ public void initializeChanceDeck() {
                 .cardType(CardType.GET_OUT_OF_JAIL)
                 .build());
     }
+
+    public Property getPropertyById(int id) {
```

Comment:
There is no need for this, as you can just use the array index which also has much better performance.